paperqa/prompts.py

```diff
@@ -99,8 +99,8 @@
 QA_PROMPT_TEMPLATE = "Q: {question}\n\nOptions:\n{options}"
 EVAL_PROMPT_TEMPLATE = (
     "Extract the single letter answer from the following question and answer"
-    "\n\n{qa_prompt}"
-    "\n\n{qa_answer}"
+    "\n\nQuestion: {qa_prompt}"
+    "\n\nAnswer: {qa_answer}"
     "\n\nSingle Letter Answer:"
```

Comment:
I'd suggest something like:
```
"Given the following question and a proposed answer to the question, return the single-letter choice that matches the proposed answer."
"\n\nQuestion: ..."
"\n\nProposed Answer: ..."
```

If I didn't know the context of this method, as a human, I'd find the original prompt unclear.

Reply:
Yeah it's a nice suggestion, but making this change breaks two of our test cases. Let's save this for another PR